ASP.NET API/WAVC_WebApi/Controllers/ApplicationUserController.cs

```diff
@@ -1,124 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IdentityModel.Tokens.Jwt;
-using System.Linq;
-using System.Security.Claims;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Identity;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Options;
-using Microsoft.IdentityModel.Tokens;
-using WAVC_WebApi.Data;
-using WAVC_WebApi.Models;
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
There's no use in having empty controller. If there's nothing there, please delete it.